### DISCUSSION TITLE
Dunfell support for R32.7.2

### DISCUSSION THREAD

**Discussion by MikeLoder** [OE4T/meta-tegra#979] [General]
Do you have plans to update the Dunfell branch for R32.7.2 to support the new Xavier AGX memory changes?

**Comment by ichergui** [comment]
Hello @MikeLoder 
I'm planning to check the diff between `L4T R32.7.1` and `L4T R32.7.2` over the weekend.
I will keep you posted

**Comment by madisongh** [comment]
@MikeLoder if you're talking about support for the 64GB AGX Xavier (and the 16GB Xavier NX), that's present in R32.7.1, too.  There are no release notes for R32.7.2 that I can find, but in comparing the contents against R32.7.1, it looks like only difference is the addition of security fixes for cboot as reported in [NVIDIA's April 2022 security bulletin](https://nvidia.custhelp.com/app/answers/detail/a_id/5343).

**Reply by madisongh** [reply]
It also appears that NVIDIA has not added the R32.7.2 packages to their debian package feeds, either, so we can't really update all the recipes to 32.7.2.  If they do that at some point, we can upgrade.  However, I suspect they didn't because the content is identical to 32.7.1.

For now, though, I've updated the cboot recipes in master and dunfell to the new version, to get the security fixes.

**Reply by ichergui** [reply]
Thanks @madisongh for the updates

**Reply by MikeLoder** [reply]
I am talking about the updated Jetson AGX Xavier memory to support SK Hynix DRAM and Western Digital eMMC. It relates to PCN208560. The change notices indicates we need R32.7.2 or newer. Thanks for looking at this.

**Reply by madisongh** [reply]
Ah, OK, I didn't notice that PCN.  Well, I'll have to see if we can extract the relevant bits from the R32.7.2 L4T kit while leaving the rest of the recipes that use the debian packages at 32.7.1.  I think it's doable, and will work on it later this week when I have some time.

**Reply by MikeLoder** [reply]
Thats great, appreciate all the hard work done on this repo its been a huge help.

**Reply by madisongh** [reply]
Looks like NVIDIA has finally published the R32.7.2 packages on their debian feeds, so I've resurrected the update work I started earlier.  I'll drop it in master first, then dunfell will follow.

**Reply by ichergui** [reply]
@madisongh please let me know if you need help

**Comment by madisongh** [comment]
The `master`, `kirkstone`, and `dunfell` branches have all been updated to R32.7.2

**Reply by MikeLoder** [reply]
Great news thanks for the effort!
